crates/xtask-bump-check/src/xtask.rs

```diff
@@ -377,6 +377,7 @@ fn check_crates_io<'a>(
                 "`{name}@{current}` needs a bump because its should have a version newer than crates.io: {:?}`",
                 possibilities
                     .iter()
+                    .map(|s| s.as_summary())
```

Comment:
Thinking on this more, should we have done these as `as_candidate` and `into_candidate`.  Currently, we should only be exposing candidates and this allows us to update the registry to expose more things without updating every caller.

Reply:
For now, aren't they filtered out?  I assumed part of the change with this is you can make more or fewer things candidates.

Reply:
That may end up being a better plan. I will keep it in mined.